plugins/kubernetes/test/models/kubernetes/cluster_test.rb

```diff
@@ -128,6 +128,20 @@ def stub_response
     end
   end
 
+  describe '#server_version' do
+    before do
+      stub_request(:get, 'http://foobar.server/version').to_return(body: '{"gitVersion": "v1.6.0"}')
+    end
+
+    it 'caches the clusters server version and returns Gem::Version object' do
```

Comment:
adding `and` to a test is always a good sign for "this should be 2 tests"

testing caching: usually more stable to call it twice and make sure the request was done only once

prefer asserting that the request was made to not get blindsided by caching

```
assert_request :get, foo, to_return: {body: 'xxx'} do
```